src/blocks/focused_window.rs

```diff
@@ -11,7 +11,9 @@ use crate::blocks::{Block, ConfigBlock, Update};
 use crate::config::SharedConfig;
 use crate::errors::*;
 use crate::scheduler::Task;
-use crate::util::escape_pango_text;
+// use crate::util::escape_pango_text;
```

Comment:
What's going on here?

Reply:
Sorry, that's no longer being used and I commented it out rather that remove it. Fixed.

Reply:
We still need to escape the text, don't we? The title/marks might have pango characters in them.

Reply:
Fixed